mne/evoked.py

```diff
@@ -434,7 +434,7 @@ def plot_image(self, picks=None, exclude='bads', unit=True, show=True,
     def plot_topomap(self, times=None, ch_type='mag', layout=None, vmin=None,
                      vmax=None, cmap='RdBu_r', sensors=True, colorbar=True,
                      scale=None, scale_time=1e3, unit=None, res=64, size=1,
-                     format="%3.1f", time_format='%01d ms', proj=False,
+                     fmt="%3.1f", time_format='%01d ms', proj=False,
```

Comment:
we can't do this without a proper deprecation warning. Users are still using this